packages/common/src/ee/AiAgent/schemas/visualizations/tableViz.ts

```diff
@@ -39,11 +40,17 @@ export const tableVizConfigSchema = z
 
 export type TableVizConfigSchemaType = z.infer<typeof tableVizConfigSchema>;
 
-export const metricQueryTableViz = (
-    vizConfig: TableVizConfigSchemaType,
-    filters: Filters,
-    maxLimit: number,
-): AiMetricQueryWithFilters => ({
+export const metricQueryTableViz = ({
+    vizConfig,
+    filters,
+    maxLimit,
+    additionalMetrics,
+}: {
+    vizConfig: TableVizConfigSchemaType;
+    filters: Filters;
+    maxLimit: number;
+    additionalMetrics?: AdditionalMetric[] | null;
```

Comment:
can we do this instead?

```suggestion
    additionalMetrics: AdditionalMetric[] | null;
```